indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -277,18 +277,15 @@ def _get_user_data(self, user):
         user_data['title'] = get_title_uuid(self.regform, user_data['title'])
         return user_data
 
-    def _process(self):
-        form = make_registration_form(self.regform, management=True)()
-        if form.validate_on_submit():
-            data = form.data
-            session['registration_notify_user_default'] = notify_user = data.pop('notify_user', False)
-            create_registration(self.regform, data, management=True, notify_user=notify_user)
-            flash(_('The registration was created.'), 'success')
-            return redirect(url_for('.manage_reglist', self.regform))
-        elif form.is_submitted():
-            # not very pretty but usually this never happens thanks to client-side validation
-            for error in form.error_list:
-                flash(error, 'error')
+    def _process_POST(self):
+        schema = make_registration_schema(self.regform)()
+        form = parser.parse(schema)
+        session['registration_notify_user_default'] = notify_user = form.pop('notify_user', False)
+        create_registration(self.regform, form, management=True, notify_user=notify_user)
+        flash(_('The registration was created.'), 'success')
+        return redirect(url_for('.manage_reglist', self.regform))
```

Comment:
maybe we can already start returning a JSON response here? redirecting an AJAX request is usually not a good idea anyway